ClientSide/life_bird/src/Components/Footer/footer.css

```diff
@@ -0,0 +1,54 @@
+.footer{
+    width: 100%;
+    height:400px;
+    border: 1px solid black;
+  
+}
+.footer{
+   width: 100%;
+   height:300px;
+   background-color: black;
+   display: flex; 
+   margin-top: 5px; 
+}
+ .logos img{
+    height:50px;
+    width:50px;
+    border-radius: 50px;
+    margin-top: 10px;
+ }
+ .logo1{
+    display: flex;
+ }
+  .facebook{
+    margin-left: 150px;
+ }
+ .insta{
+    margin-left: 20px;
+ }
+ .twitter{
+    margin-left: 20px;
+ } 
+ .h1{
+    color:whitesmoke;
+ }
+ 
+ .p{
+    color:whitesmoke;
+ }
+ .location{
+    height:200px;
+    width:500px;
+    margin-left: 200px;
+    margin-top: 50px;
+ }
+ .logos{
+    height:200px;
+    width:500px; 
```

Comment:
please maintain same class name for common css properties

Reply:
i used same class names